test/fulfil/client_test.rb

```diff
@@ -68,5 +68,14 @@ def test_do_not_retry_when_retry_is_disabled
         end
       end
     end
+
+    def test_put_that_returns_array_wrapped_integer
+      stub_fulfil_put('sale.sale/1/copy', 'sale_sale_copy', '[]')
+
+      client = Fulfil::Client.new
+      order_count = client.put(model: 'sale.sale', id: 1, endpoint: 'copy', body: [])
```

Comment:
I don't think the naming is accurate here. 

```suggestion
      order_copy = client.put(model: 'sale.sale', id: 1, endpoint: 'copy')
```

Reply:
Cool I updated those but removing the body causes failures.